app/src/lang/translations/en-US.yaml

```
@@ -463,13 +463,16 @@ open_file_in_tab: Open file in new tab
 start_export: Start Export
 not_available_for_local_downloads: Not available for local downloads
 exporting_all_items_in_collection: Exporting all items ({total}) within {collection}.
-exporting_limited_items_in_collection: Exporting {limit} out of {total} items within {collection}.
-exporting_no_items_to_export: No items to export. Adjust the exporting configuration.
+exporting_limited_items_in_collection: Exporting {count} out of {total} items within {collection}.
+exporting_no_items_to_export: No items to export. Adjust the exporting configuration and/or add items to the collection.
 exporting_download_hint: Once completed, this {format} file will automatically be downloaded to your device.
 exporting_batch_hint: This export will be processed in batches, and once completed, the {format} file will be saved to the File Library.
 exporting_batch_hint_forced:
   Due to the large number of items, this export must be processed in batches, and once completed, the {format} file will
   be saved to the File Library.
+exporting_query_limit_hint_forced:
+  Due to the number of items exceeding the query limit, this export must be processed in batches, and once completed, the {format} file will
```

for my taste the sentence is a bit too long.
 ```suggestion
  Due to the number of items exceeding the query limit, this export must be processed in batches. Once completed, the {format} file will
```

---

Also wonder if there is a better way to explain to the end-user that this is due to the query limit.
Perhaps @phazonoverload could help with the wording of these sentences? 😇

---

@paescuj I don't think we have to explain in-app _why_ it's done in batches

---

Simplified to:
https://github.com/directus/directus/blob/3cb97c3d91aef0e46f4f75a472faa1e0657f51bc/app/src/lang/translations/en-US.yaml#L473-L475